infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogFiles.java

```diff
@@ -911,6 +914,11 @@ LogEntry getLogEntry(long address, LogData entry) {
                 new UUID(rank.getUuidMostSignificant(), rank.getUuidLeastSignificant()));
     }
 
+    private Set<AddressMetaData> writeRecord(SegmentHandle fh,
+                                             List<LogData> entry) throws IOException {
+        return null;
```

Comment:
What's the point of this method?

Reply:
should have marked this as WIP, but I'm implementing batch writing, where consecutive addresses can be written to storage in one operation. Its needed to speed up catch up when adding a new node.